### PR TITLE
Add an option to modify timeout for namespace duration in e2e Framework

### PR DESCRIPTION
**Pull request by gmarek**
cc @quinton-hoole @jszczepkowski

### CODE REVIEW

File: test/e2e/load.go [hunk 1]
```diff
@@ -53,7 +53,7 @@ var _ = Describe("Load capacity", func() {
 	var nodeCount int
 	var ns string
 	var configs []*RCConfig
-	framework := Framework{BaseName: "density"}
+	framework := Framework{BaseName: "density", NamespaceDeletionTimeout: 10 * time.Minute}
```
Comment (wojtek-t) [inline test/e2e/load.go]: 1 hour please

**Comment by wojtek-t** [comment]
Just 1 minor comment

**Comment by gmarek** [comment]
Done.

**Comment by wojtek-t** [comment]
LGTM - thanks for the fix!

**Pull request merged**